### ISSUE TITLE
Exception @ SimpleLapsGui.ps1:148 / Null reference Exception

### ISSUE THREAD

**Issue by F1reF0x** [htcfreek/SimpleLapsGui#10]
## 📝 Describe the bug

When searching for a computer, the first entry in the password history listview is generating an error

```
Ausnahme beim Aufrufen von "Add" mit 1 Argument(en):  "Der Objektverweis wurde nicht auf eine Objektinstanz
festgelegt."
In C:\Users\username\Downloads\SimpleLapsGui.ps1:148 Zeichen:13
+             $item.SubItems.Add($pwds[$i].Password)
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
    + FullyQualifiedErrorId : NullReferenceException`
```

## Version information

- Simple LAPS GUI:
- Operating System:

## ✔ Expected behavior

No Exception

## Screenshots and additional context

A possible fix you can find here [https://gist.github.com/F1reF0x/851bcbe58e2c4e59bc163efd3a06811b](url)

**Comment by htcfreek** [comment]
@F1reF0x 
What specific search result is triggering the crash? What is the reason?

**Comment by F1reF0x** [comment]
Just the normal Workstation name .. I realized, that the first password / element is null and throws an exception. Especially when the powershell console is visible you'll see the exception. With this small fix an element which is null will be for example none and no expetion is thrown. It's not really a bug - justr a visibility thing.

**Comment by F1reF0x** [comment]
![image](https://github.com/htcfreek/SimpleLapsGui/assets/4030053/e56f17b3-1408-4702-bcc6-3b5c645085b2) Here is an example - fresh LAPS activated AD - OU with a fresh client as test - the first entry of the history was null (don't know why) and threw the exception.

**Comment by htcfreek** [comment]
Interesting. It's the first time I see this. The client never had a password before or did you switch from Legacy LAPS to Windows LAPS?

(I personally think this should be reported to MS. I "null" password in the history makes no sense.)

**Comment by F1reF0x** [comment]
Hope this will work - [https://gist.github.com/F1reF0x/851bcbe58e2c4e59bc163efd3a06811b](url)

**Comment by htcfreek** [comment]
Fixed and will be included in version `1.2`.

**Comment by htcfreek** [comment]
Fixed and fix released with version `1.2`.
